custom/_legacy/mvp_docs/pillows.py

```diff
@@ -10,7 +10,6 @@
 from pillowtop.listener import BasicPillow
 
 pillow_logging = logging.getLogger("pillowtop")
-pillow_eval_logging = logging.getLogger("pillowtop_eval")
```

Comment:
If I understand correctly, the difference between these loggers is that pillowtop_eval would send things above INFO level to sentry (and only sentry).  Without sentry, I believe this will now pickup errors in the pillowtop handler.